bridgerelayer/chains/substrate/streamer.go

```diff
@@ -118,33 +118,40 @@ func (ss *Streamer) SubscribeBlocks(blockRetryLimit int) error {
 
 			events := Events{}
 			err = records.DecodeEventRecords(&ss.Core.MetaData, &events)
-			log.Info("DecodeEvents")
-			if err != nil {
-				log.Error("Error decoding event", err)
-				return err
-			}
 
-			ss.handleEvents(&events)
+			// TODO: DecodeEventRecords will fail if the chain emits events from pallet_scheduler.
+			// We'll need to decode those events properly by adding the necessary support in ./events.go.
+			// For now we just ignore the error since pallet_scheduler events aren't important for us.
```

Comment:
This will lead to bugs, unfortunately. The events are decoded in order, and if a pallet_scheduler event comes before one of the relevant events, the whole block will be skipped. Please create a ticket so we fix this as soon as GSRPC has been updated.

Reply:
Yeah this a critical issue. I've cut this ticket to track:  https://github.com/Snowfork/polkadot-ethereum/issues/60